# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/reiserfsprogs/hotfix-gcc14.patch.musl
# Copyright (C) 2024 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- reiserfsprogs-3.6.27/resize_reiserfs/resize_reiserfs.c.vanilla	2024-07-25 15:32:31.570673833 +0200
+++ reiserfsprogs-3.6.27/resize_reiserfs/resize_reiserfs.c	2024-07-25 15:32:41.286673716 +0200
@@ -14,6 +14,7 @@
 
 #include "resize.h"
 #include <limits.h>
+#include <libgen.h>
 
 static int opt_banner = 0;
 static int opt_skipj = 0;
--- reiserfsprogs-3.6.27/resize_reiserfs/fe.c.vanilla	2024-07-25 15:33:00.686673482 +0200
+++ reiserfsprogs-3.6.27/resize_reiserfs/fe.c	2024-07-25 15:33:09.110673380 +0200
@@ -4,6 +4,7 @@
  */
 
 #include "resize.h"
+#include <sys/mount.h>
 
 /* the front-end for kernel on-line resizer */
 int resize_fs_online(char *devname, long long int blocks)
